docs/Users_Guide/tc-diag.rst

```diff
@@ -152,7 +149,21 @@ Configuring Data Censoring and Conversion Options
   censor_val    = [];
   convert(x)    = x;
 
-These data censoring and conversion options are common to multiple MET tools and are described in :numref:`config_options`. They can be specified separately in each **data.field** array entry, described below. If provided, those operations are performed after reading the gridded data but prior to converting to the cylindrical coordinate range-azimuth grid.
+These data censoring and conversion options are common to multiple MET tools and are described in :numref:`config_options`. They do not actually appear in the default configuration file but can be specified separately in each **data.field** array entry, described below. If provided, those operations are performed after reading the gridded data but prior to converting to the cylindrical coordinate range-azimuth grid.
```

Comment:
In what case would a user ever regrid the data read in before converting to the cylindrical coordinate range-azimuth grid? The only thing I can think of is if one wanted to downgrade the resolution of a high-res model to the GFS before doing the transformation in order to do a "fair" comparison. But I can't think of any other reasons. Would it be good to put a note about this?

Reply:
This is a mistake. I see that the "Configuring regridding options" section appears twice when it should only appear once. I'll delete the second one.

This regridding setting controls how regridding to cylindrical coordinates is done and nothing else. There is no option to do what you mentioned... reading high-res, regrid to low-res, regrid to cylindrical coordinates.

Reply:
I'm updating the description to state:
```
The **regrid** dictionary is common to multiple MET tools and is described in :numref:`config_options`. It specifies how the input data should be regridded to cylindrical coordinates prior to compute diagnostics. It can be specified separately in each **data.field** array entry, described below. The default setting uses nearest neighbor interpolation for all fields.
```